broker/src/main/java/io/camunda/zeebe/broker/partitioning/PartitionAdminAccess.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.broker.partitioning;
+
+import io.camunda.zeebe.util.sched.future.ActorFuture;
+
+public interface PartitionAdminAccess {
```

Comment:
Happy to receive help on this one, or defer it to a follow up issue.

a) I have no idea what the contract is
b) I don't even know what a contract can look like in an Actor world, because basically anything I can guarantee is that I will send a message to an actor. What the actor does with the message is then an implementation detail which I cannot really guarantee. But maybe I am overcomplicating things here.

Reply:
It's not about `PartitionAdminAccess` as an actor, but rather - what is it used for? What are its responsibilities? If I call `pauseExporting`, what happens? We know what happens - if the operation is successful, exporting is stopped, and this is persisted across restarts, such that no exporting will take place for this partition on this node until `resumeExporting` is called. It's also safe to call multiple times.

I'm happy to write this in a follow up PR though, especially since here you just extracted this, but in general I would like that we document public interfaces.